src/libs/parsers/formatDates.ts

```diff
@@ -112,7 +112,7 @@ export function joinArrayElement(array: (string | number)[] | MonthDays) {
     return array.join(' et ')
   } else {
     const first = array.slice(0, -1).join(', ')
-    const last = array.slice(-1)
+    const last = array.slice(-1).join('')
```

Comment:
Si on veut récupérer le dernier élément du tableau on pourrait juste faire : 

```suggestion
    const last = array.at(-1)
```

Reply:
`at` ne réagit pas comme `slice`, il peut être `undefined` donc je ne peux pas appliquer le `join('')` pour avoir un string. Sinon il faut que j'utiliser `String()` pour éviter l'erreur eslint

Reply:
Je vois. Sinon tu peux rajouter : `const last = array.at(-1) ?? ''` et ça devrait faire taire eslint je pense